src/svc-nifid.yml

```diff
@@ -11,6 +11,8 @@ paths:
     $ref: './svc/nifid/paths/broker_subs_id.yml'
   /broker/subs/{id}/status:
     $ref: './svc/nifid/paths/broker_subs_id_status.yml'
+  /broker/subs/statuses:
+    $ref: './svc/nifid/paths/broker_subs_status.yml'
```

Comment:
I think you need to add the new schemas and request body change in this file then it should fix the auto generated ones in contracts